recipe/meta.yaml

```
@@ -56,12 +61,15 @@ outputs:
   - name: {{ name }}
     build:
       script: python -m pip install .[empy,report-timings] --no-deps --ignore-installed -vv
```

```suggestion
      script: {{ PYTHON }} -m pip install .[empy,report-timings] --no-deps --ignore-installed -vv
```

I think they prefer this now...

---

For some reason that doesn't work in the `outputs` section.

---

(changed here - https://github.com/conda-forge/cylc-flow-feedstock/pull/29/files)

---

Okay, pls ignore :)